### PR TITLE
[CELEBORN-1302] Introduce overview module for dashboard frontend

### PR DESCRIPTION
**Pull request by tiny-dust**
### What changes were proposed in this pull request?

Introduce overview module for dashboard frontend:

image

### Why are the changes needed?

Dashboard frontend should support overview page to display the overview details of Celeborn.

### Does this PR introduce _any_ user-facing change?

no.

### How was this patch tested?

Local test.

### CODE REVIEW

**Review by labbomb — APPROVED** [review]
LGTM. Thanks for your contribution, @SteNicholas PTAL.

File: web/src/views/overview/components/WorkerOverview.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+<template>
+  <n-card hoverable>
+    <n-descriptions label-placement="left" title="Worker State Nums">
+      <n-descriptions-item label="Decommission"> {{ data.decommissionNum }} </n-descriptions-item>
```
Comment (SteNicholas) [inline web/src/views/overview/components/WorkerOverview.vue:22]: Add the normal num in `Worker State Nums`:
```
<n-descriptions-item label="Normal"> {{ data.normalNum }} </n-descriptions-item>
```

File: web/src/views/overview/components/StorageOverview.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+<template>
+  <n-card hoverable>
+    <n-descriptions label-placement="left" title="Storage Usage Infos">
+      <n-descriptions-item label="disk"> {{ data.diskUsage }} </n-descriptions-item>
```
Comment (SteNicholas) [inline web/src/views/overview/components/StorageOverview.vue:22]:
```suggestion
      <n-descriptions-item label="Disk"> {{ data.diskUsage }} </n-descriptions-item>
```

File: web/src/views/overview/components/StorageOverview.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <n-descriptions-item label="memory">
```
Comment (SteNicholas) [inline web/src/views/overview/components/StorageOverview.vue:23]:
```suggestion
      <n-descriptions-item label="Memory">
```

File: web/src/views/overview/components/StorageOverview.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        {{ data.memoryUsage }}
+      </n-descriptions-item>
+      <n-descriptions-item label="hdfs">
```
Comment (SteNicholas) [inline web/src/views/overview/components/StorageOverview.vue:26]:
```suggestion
      <n-descriptions-item label="HDFS">
```

File: web/vite.config.ts [hunk 5]
```diff
@@ -25,6 +25,18 @@ import { NaiveUiResolver } from 'unplugin-vue-components/resolvers'
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  // FIXME: Get the real service address and then change it
```
Comment (SteNicholas) [inline web/vite.config.ts:28]:
```suggestion
  // TODO: Get the backend service address and then change this uri.
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
@tiny-dust, thanks for the contribution of overview module. Should the `tenant/index.ts` update the `export`?

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request closed without merging**